### PR TITLE
Add sourceview

### PR DESCRIPTION
**Pull request by GuillaumeGomez**

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -13,6 +13,7 @@ layout: wide
 [GLib]: https://developer.gnome.org/glib/stable/
 [GTK]: https://developer.gnome.org/gtk3/stable/
 [Cairo]: http://cairographics.org/documentation/
+[GtkSourceView]: https://wiki.gnome.org/Projects/GtkSourceView
```
Comment (EPashkin) [inline index.md]: This won't do anything without using it :(

Comment (GuillaumeGomez) [inline index.md]: What do you mean?

Comment (EPashkin) [inline index.md]: As I understand this just footnote. Others used in "bindings for"

Comment (GuillaumeGomez) [inline index.md]: Ah right, well, I didn't put the doc for the sourceview yet. I'm going to do it and add sourceview into the documented crates at the same time.

**Comment by EPashkin** [comment]
By way about docs:
It normal that cairo docs don't contains `create_from_png` ?
Or doc generator don't detect feature-gated modules?

**Comment by GuillaumeGomez** [comment]
I think it doesn't. In the future we'll need to generate docs more carefully in order to have every features for all crates. (yep, it's quite annoying)

**Pull request merged**